spec/mailers/alert_mailer_spec.rb

```diff
@@ -8,7 +8,7 @@
   let(:body) { mail.body.raw_source }
   let(:subscription) do
     subscription = create(:daily_subscription, email: 'an@email.com',
-                                               reference: 'a-reference',
+                                               frequency: 'daily',
                                                search_criteria: {
                                                  subject: 'English',
```

Comment:
In Cesi's PR, he rightly corrects 'subject' to 'keyword' here as we don't filter by subject (we used to, a year ago).